chore: Use text value comparison as default option for body verification rules

e-fisher opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description

From internal feedback: in verification rule, comparing response body to recording could lead to massive script, we still want to provide this option, but not as a default when `Response body` is selected as `Target`. Instead, select `Text value` as default.

## How to Test

1. Create a new verification rule
2. Change `Target` to `Response body`
3. Verify `Text value` option is pre-selected.


## Checklist

- [x] I have performed a self-review of my code.
- [ ] I have added tests for my changes.
- [ ] I have run linter locally (`npm run lint`) and all checks pass.
- [ ] I have run tests locally (`npm test`) and all tests pass.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
<!-- - [ ] Any other relevant item -->

## Screenshots (if appropriate):

![screencapture 2025-03-24 at 13 59 08](https://github.com/user-attachments/assets/a5333824-7519-406c-85e2-f3120b4c95e9)


## Related PR(s)/Issue(s)

<!-- - <https://github.com/grafana/k6-studio/issues/...> -->

<!-- Does it resolve an issue? -->

<!-- Resolves #ISSUE-ID -->

<!-- Thanks for your contribution! 🙏🏼 -->

going-confetti reviewed (commented):
I have two thoughts:
1. We usually remove chores from the release notes, so maybe this should be a `feat` or a `fix`? Though I can see how none of them really fits here.
2. IMO it would make more sense to use `includes` rather than `equals` - it's hard to imagine someone copy-pasting entire body into a small text input (which is an issue worth thinking about on its own - maybe it should use textarea/monaco 🤔)

going-confetti reviewed (approved):
LGTM!

e-fisher commented:
1. Good point on chores being excluded from release notes, I've changed to `fix`. Although it feels like we have a need for an additional label, an "improvement" that's neither a fix nor new feature 🤔 
2. Agree, I've updated to use "contains" as default. A use case for equals would be verifying JSON body, e.g., `{"success":true}`. We'll need to revisit this when we do rule form's revamp, a monaco editor could work here, if we add support for both html and JSON highlighting.
